kitsune/users/api.py

```diff
@@ -346,6 +348,39 @@ def generate(self, request, **kwargs):
             'token': token.key,
         })
 
+    # This is routed to /api/2/user/weekly-solutions/
+    def weekly_solutions(self, request, **kwargs):
+        """
+        Return the most helpful users in the past week.
+        """
+        start = datetime.now() - timedelta(days=7)
+        # Get a list of users and the number of solutions they have in the last week.
+        # It looks like [{'creator__username': 'bob', 'creator__count': 12}, ...]
+        # This uses ``username`` instead of ``id``, because ``username`` appears
+        # in the output of ``ProfileFKSerializer``, whereas ``id`` does not.
+        raw_counts = (
+            Answer.objects
+            .exclude(solution_for=None)
+            .filter(created__gt=start)
+            .values('creator__username')
+            .annotate(Count('creator'))
+            )
```

Comment:
If this ends up being slow, you can use the contribution data. It should be very similar to this but filtering solutions only: https://github.com/mozilla/kitsune/blob/master/kitsune/community/utils.py#L17